doc/src/bond_over_grpc.md

```diff
@@ -230,23 +230,39 @@ The proxy stub can then be used to make calls to the server as follows:
 ExampleRequest request;
 // Fill in request fields here
 
-bond::ext::gRPC::wait_callback<ExampleResponse> cb;
-client.AsyncExampleMethod(request, callback);
+// Blocking version using std::future
+try
+{
+    ExampleResponse response = client.AsyncExampleMethod(request).get().response().Deserialize();
+    // Examine response here
+}
+catch (const bond::ext::gRPC::UnaryCallException& e)
+{
+    // Examine e.status().error_message()
```

Comment:
I would leave this as `e.status()` instead of implyting that strings need to be parsed.
Same thing below.